README.md

```diff
@@ -10,6 +10,63 @@ Neural Speed is an innovation library designed to provide the efficient inferenc
 
 > Neural Speed is under active development so APIs are subject to change.
 
+## Quick Start
+There are two methods for utilizing the Neural Speed:
+- [Transformer-based API](#How-to-use-Transformer-based-API)
```

Comment:
link back to ITREX and remove "How to use: Transformer-based API" in NeuralSpeed

Reply:
link to supported models when use_llm_runtime= True